python/ray/tune/trial.py

```diff
@@ -177,6 +177,28 @@ def has_trainable(trainable_name):
         ray.tune.registry.TRAINABLE_CLASS, trainable_name)
 
 
+def _find_newest_ckpt(ckpt_dir):
+    try:
```

Comment:
Can you add a case if this is None (and not rely on the code path failure to catch)?